puppetserver/Dockerfile

```diff
@@ -91,19 +91,21 @@ RUN chmod +x /docker-entrypoint.sh /healthcheck.sh /docker-entrypoint.d/*.sh &&
 # release (build from packages)
 ######################################################
 
-FROM base as release
+FROM base AS release
 
 ARG PUPPET_RELEASE=8
 ARG PUPPETSERVER_VERSION=8.6.1
 ARG UBUNTU_CODENAME
+ARG PUPPET_USER_UID=999
```

Comment:
I got that id from logging onto a running instance of the container. 
I did not contact puppet as i think setting fixed id's in a package install is a very bad idea.

Reply:
And, if puppet has/provides a way to specify a uid/gid during install, that still would have to be done in the Dockerfile. So this is certainly less intrusive.

Reply:
it's a static id. on my server it's to 999. i'm okay with it

```
id puppet
uid=999(puppet) gid=999(puppet) groups=999(puppet)
```

Reply:
I don't think it's static. You just get a random free id:

```
$ id puppet
uid=110(puppet) gid=116(puppet) groups=116(puppet)
```

```
# id puppet
uid=106(puppet) gid=112(puppet) groups=112(puppet)
```

```
# id puppet
uid=997(puppet) gid=998(puppet) groups=998(puppet)
```

Reply:
hmm strange. in de source it says something about uid 52

https://github.com/puppetlabs/puppetserver/blob/main/project.clj#L95-L98

Reply:
so what do we do now. is it okay, or do we have do something?

Reply:
As this is a 'controlled' environment, it seems ok to me to set a fixed UID/GID.
If there would ever be a problem with this, which i doubt it would, it will manifest itself at build time, so that can be fixed without affecting users.

Reply:
i agree.

Reply:
i asked in the slack about, but will continue here with the merge. if there comes up something new, we can patch it later :)